src/troubleshooting.rst

```diff
@@ -466,6 +471,66 @@ environments. If you encounter this issue in combination with containers,
 please let us know.
 
 
+Understanding Existing Workflows
+--------------------------------
```

Comment:
This all looks good to me, but I don't think the heading makes much sense.

This bit could be expanded into "Cylc debugging utilities" covering:
* validate
* lint
* view
* config

Reply:
Re heading, maybe "Debugging workflow configurations"?

Reply:
(see also the cheat sheet which now covers this too)